spec/ransack/adapters/active_record/base_spec.rb

```diff
@@ -145,6 +145,9 @@ def self.sane_adapter?
           end
 
           it "should function correctly with a multi-parameter attribute" do
+            ::ActiveRecord::Base.default_timezone = :utc
+            Time.zone = 'UTC'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.